src/js/geo-dash.js

```diff
@@ -1729,6 +1730,7 @@ export function pageInit(args) {
     ReactDOM.render(
         <GeoDashNavigationBar
             userName={args.userName || "guest"}
+            plotId={args.plotId || "0"}
```

Comment:
- For the most part, CEO uses -1 to mean not a valid selection / value instead of 0.  Here 0 would not be a valid plotid if it was generated by us, but it could be from external data if the user was using a 0 based index.
-   plotId is an integer.
`plotId={parseInt(args.plotId) || -1}`